src/upser-defaults/upser-defaults.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable, OnModuleInit } from '@nestjs/common';
+import { InjectConnection } from '@nestjs/mongoose';
+import { Connection } from 'mongoose';
+
+@Injectable()
+export class UpserDefaultsService implements OnModuleInit {
+  constructor(@InjectConnection() private readonly connection: Connection) {}
+
+  async onModuleInit(): Promise<void> {
+    await this.upsertDefaults();
+  }
+
+  async upsertDefaults(): Promise<void> {
+    const dbName = 'dev';
+    const db = this.connection.useDb(dbName, { useCache: true });
+
+    const collections = await db.db.listCollections().toArray();
+
+    if (!collections.some((collection) => collection.name === 'users')) {
```

Comment:
kolekcja zmieniłą nazwę btw, ale to jak zmodyfikujesz na arraykę to nei bedzie problemu, zrebejsuj na developa i sprawdź linta / prettiera też